fdbclient/include/fdbclient/TenantManagement.actor.h

```diff
@@ -509,8 +523,13 @@ Future<Void> renameTenantTransaction(Transaction tr, TenantNameRef oldName, Tena
 }
 
 ACTOR template <class DB>
-Future<Void> renameTenant(Reference<DB> db, TenantName oldName, TenantName newName) {
+Future<Void> renameTenant(Reference<DB> db,
+                          TenantName oldName,
+                          TenantName newName,
+                          Optional<int64_t> tenantId = Optional<int64_t>(),
+                          ClusterType clusterType = ClusterType::STANDALONE) {
```

Comment:
@sfc-gh-ajbeamon does this function need to take into consideration `configurationSequenceNum` too?

Reply:
I was thinking about this too. We don't currently use it with a sequence number, so it's certainly not required. However, if we don't allow specifying one then this function will only work on standalone clusters and could remove that argument as well.

I'd say it's fine to keep it simple for now and if we need to extend it later we can add the arguments we need.